client/src/modules/creditors/groups/creditor-groups-list.html

```diff
@@ -19,7 +19,7 @@ <h5 style="margin-top : 0px;"></span><span translate> FORM.LABELS.ACCOUNT_NUMBER
               </div>
             </div>
 
-            <span class="fa fa-bars"></span> <b>{{ cg.total_creditors }}</b> {{ 'FORM.LABELS.CREDITOR_SUBCRIBED' | translate }}
+            <span class="fa fa-bars"></span> <b>{{ cg.total_creditors }}</b> <span transalet>FORM.LABELS.CREDITOR_SUBCRIBED</span>
```

Comment:
It looks like there is a typo here - this should be `translate` not `transalet`.